### DISCUSSION TITLE
Deploy with Windows Group Policy

### DISCUSSION THREAD

**Discussion by kempert** [immense/Remotely#949] [Q&A]
Hello, 

is there anywhere a way, to deploy the Remotely script by an Group Policy on Windows, to rollout the agent to alle Devices in the Domain?
All tests by myself, with an startup script, are not successed.

Greetings

**Comment by CMcNichol-GBSTP** [comment]
Hi @kempert 

I have this implemented in my environment using a GPO, in my environment, I use multiple GPOs to add the devices into their respective Device Groups.
I did have to edit the script slightly from what remotely gives, but this then installs without issues.

Did you check your GPO logs, and attempt to install manually from PowerShell to check it is also working?


Chris

**Reply by kempert** [reply]
Hi,

Thanks for your reply. I've not checked the GPO Logs, should i Check them on the DC oder Local? 

Could you show the Script you are using? Is the Script signed? 

Greetings

**Comment by CMcNichol-GBSTP** [comment]
Hi,

From your Group Policy Management console, you can get the group policy results for the computer that isn't installing which can confirm that the GPO is correctly applying. On the PC, you should find the remotely install log in C:\Windows\Temp\Remotely_Install.txt which will also list any errors you may have - feel free to share the log and I'll attempt to help you out with it.

I will dig the script out and confirm, I believe it is generally the same as the main install script, but I removed some parameters like server url, and hard-coded these into the script.
I did also sign my script as I have enforced signed PowerShell scripts in my environment.


Chris
